content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -98,6 +98,54 @@ appear. In this example, only :guilabel:`Marc Demo` meets these criteria.
    :align: center
    :alt: The employees with expert Javascript skills.
 
+Use case: Assess highest improvement
+====================================
+
+Another way to modify the :guilabel:`Appraisal Skills Report` is to identify the employee who has
+the highest amount of improved skills over a specific period of time. In this example, the employee
+with the most amount of improved skills for the third quarter is being determined.
+
+To view this information, first remove the default filter in the search bar. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Improvement` beneath the :icon:`fa-filter` :guilabel:`Filters` column. Enabling this
+filter only presents skills that have improved.
+
+With the search bar drop-down menu still expanded, click :guilabel:`Add Custom Filter` at the bottom
+of the :icon:`fa-filter` :guilabel:`Filters` column, and an :guilabel:`Add Custom Filter` pop-up
+window appears.
+
+Select :guilabel:`Create Date` for the first drop-down field, then select :guilabel:`is between` for
+the second drop-down field. Once :guilabel:`is between` is selected, a second field appears after
+the last field. Using the calendar selector, set the first date to :guilabel:`07/01/2024` and the
+second date to :guilabel:`09/30/2024`. Once all the fields are properly formatted, click
+:guilabel:`Add`.
```

Comment:
Thank you for catching that! I tried to select 9/30 but it reverts to the current day, since I made te image on the 16th! Since today is the 30th I was able to update this pic! =)